eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -4,6 +4,7 @@ parameters:
   EnvVars: {}
   MaxParallel: 0
   BuildInParallel: true
+  ExcludeTrack1: false
```

Comment:
In general we refer to track 1 as data-plane and track 2 as client libraries so if we do end up needing to keep this we should use that naming convention.